pkg/controller/restore_controller.go

```diff
@@ -406,8 +406,8 @@ func (c *restoreController) fetchBackupInfo(backupName string, pluginManager plu
 		}
 
 		log := c.logger.WithField("backupName", backupName)
-		log.Debug("Backup not found in backupLister, checking each backup location directly, starting with default...")
-		return c.fetchFromBackupStorage(backupName, pluginManager)
+		log.Debug("Backup not found in backupLister")
```

Comment:
the godoc for this function should be updated (L399-400) -- can just say that it gets the backup API obj from the lister and a backup store for the backup's storage location